Update to 0.19 of hyperfoil

Jumping from `0.17` to `0.19` does not directly work, b/c there is a breaking API change on 0.18

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project eventing-hyperfoil-benchmark-vertx-receiver: Compilation failure
[ERROR] /home/matzew/go/src/github.com/openshift-knative/eventing-hyperfoil-benchmark/vertx-receiver/src/main/java/io/openshift/serverless/knative/eventing/hyperfoil/benchmark/VertxReceiverVerticle.java:[104,46] constructor RequestStatsMessage in class io.hyperfoil.clustering.messages.RequestStatsMessage cannot be applied to given types;
[ERROR]   required: java.lang.String,java.lang.String,int,int,java.lang.String,io.hyperfoil.api.statistics.StatisticsSnapshot
[ERROR]   found:    java.lang.String,java.lang.String,int,boolean,int,java.lang.String,io.hyperfoil.api.statistics.StatisticsSnapshot
[ERROR]   reason: actual and formal argument lists differ in length
[ERROR] 
```

/cc @rvansa

--- a/vertx-receiver/pom.xml
+++ b/vertx-receiver/pom.xml
@@ -11,7 +11,7 @@
     <properties>
         <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.target>11</maven.compiler.target>
-        <hyperfoil.version>0.17</hyperfoil.version>
+        <hyperfoil.version>0.19</hyperfoil.version>
     </properties>
 
     <dependencies>

--- a/vertx-receiver/src/main/java/io/openshift/serverless/knative/eventing/hyperfoil/benchmark/VertxReceiverVerticle.java
+++ b/vertx-receiver/src/main/java/io/openshift/serverless/knative/eventing/hyperfoil/benchmark/VertxReceiverVerticle.java
@@ -101,7 +101,7 @@ public class VertxReceiverVerticle extends BaseAuxiliaryVerticle {
           // Normally end time is capped to Statistics.endTime (which we have set prematurely)
           clone.histogram.setEndTimeStamp(clone.histogram.getStartTimeStamp() + 1000);
           log.info("Sending stats {}/{}/{} #{} ({} requests) to controller", ps.runId, ps.phaseId, ps.metric, clone.sequenceId, clone.requestCount);
-          vertx.eventBus().send(Feeds.STATS, new RequestStatsMessage(deploymentID(), ps.runId, phaseId, false, 0, ps.metric, clone));
+          vertx.eventBus().send(Feeds.STATS, new RequestStatsMessage(deploymentID(), ps.runId, phaseId, 0, ps.metric, clone));
         }
       });
     }

